client/src/pages/SignUp/SignUp.tsx

```diff
@@ -38,26 +38,25 @@ export default function Register(): JSX.Element {
 
   return (
     <Grid container component="main" className={classes.root}>
-      <Grid item xs={12} sm={8} md={7} elevation={6} component={Paper} square>
+      <Grid item xs={12} sm={8} md={7} className={classes.container}>
```

Comment:
Is there any reason we aren't using the `Paper` component here? I noticed that we're manually applying a box-shadow and styles to this component